File: SeleniumQuestion/python/test_class.py
import os

import pytest
from selenium import webdriver
from chromedriver_py import binary_path


@pytest.fixture
def before_each():
    web_driver = webdriver.Chrome(executable_path=binary_path)
    file_path = os.path.abspath(os.path.join(os.getcwd(), os.pardir, "index.html"))
    web_driver.get(file_path)
    return web_driver


class TestClass:
    def test_page(self, before_each):
        web_driver = before_each
        assert "Test me" in web_driver.title

File: SeleniumQuestion/dotnet/TestClass.cs
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System;
using System.IO;

namespace seleniumquestion
{
    public class TestClass
    {
        IWebDriver webdriver;

        [SetUp]
        public void Setup()
        {
            var options = new ChromeOptions();
            options.AddArguments("start-maximized", "no-sandbox");

            webdriver = new ChromeDriver(Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory), options);
            
            var relativePathToIndexFile = Path.Combine("..", "..", "..", "..", "index.html");
            var filePath = Path.GetFullPath(relativePathToIndexFile, AppDomain.CurrentDomain.BaseDirectory);
            webdriver.Navigate().GoToUrl(filePath);
        }

        [TearDown]
        public void Teardown()
        {
            webdriver.Quit();
        }

        [Test]
        public void TestPage()
        {

            Assert.IsTrue(webdriver.Title.Equals("Test me"));
        }
    }
}